tests/baselines/reference/project/realisticNpmTypes/amd/realisticNpmTypes.json

```diff
@@ -0,0 +1,19 @@
+{
+    "scenario": "Realistic npm package types",
+    "projectRoot": "tests/cases/projects/realisticNpmTypes",
+    "inputFiles": [
+        "index.ts"
+    ],
+    "declaration": true,
+    "baselineCheck": true,
+    "moduleResolution": "node",
+    "resolvedInputFiles": [
+        "lib.d.ts",
+        "node_modules/@types/m1/index.d.ts",
```

Comment:
Yeah, see the last line I added to the gitignore file on my prior merge to master. That nearly got me too.